mappings/net/minecraft/util/registry/DynamicRegistries.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_5458 net/minecraft/util/registry/DynamicRegistries
```

Comment:
said to be `BuiltinRegistries` as there is a `missing built-in` message at where registry tracker uses  this class

Reply:
renamed